frontend/packages/kubevirt-plugin/src/types/url.ts

```diff
@@ -0,0 +1,17 @@
+export type InitialData = {
```

Comment:
we could work on the naming a bit. What about `VMWizardInitialData` and `VMWizardBootSourceParams` ?

Reply:
sounds good, renamed all